pom.xml

```diff
@@ -29,8 +29,15 @@
       <id>pepov</id>
       <name>Peter Wilcsinszky</name>
     </developer>
+    <developer>
+      <name>Jens Mittag</name>
```

Comment:
I have no objections to maintain the plugin. I only added myself in the list because I assumed it resembles all past contributors. I have no issue with removing my name from here.